rundeckapp/grails-spa/packages/ui-trellis/src/app/components/home/tests/HomeCardList.spec.ts

```diff
@@ -1,4 +1,4 @@
-import { shallowMount } from "@vue/test-utils";
+import { shallowMount, VueWrapper } from "@vue/test-utils";
```

Comment:
IIRC correctly, these changes (both in HomeCardList and HomeView) were merged to main with another PR, but somehow when I applied the rebase they ended up becoming a separate commit.